sources/packages/forms/src/form-definitions/partnerinformationandincomeappeal.json

```diff
@@ -366,7 +366,7 @@
           "id": "eoar7nv"
         },
         {
-          "label": "I have been authorized by my partner to submit their financial information including their net income (previous tax year's return's Line 15000 information)",
+          "label": "I have been authorized by my spouse/common-law partner to submit their financial information, including their total reported income from their {{data.programYear}} tax return.",
```

Comment:
The variable program year is different here. It is not the tax year. 
![image](https://github.com/user-attachments/assets/4f273a49-4f5b-4e59-a7ea-9af24fa8a36c)

Reply:
Suggest for this we just replace the dynamic "data.programYear" with "insert year", and then leave it to next ticket to update appropriately. That work @dheepak-aot ?

Reply:
@ninosamson the idea of updating this ticket is to have only static content updates and all the content updates involving dynamic variable will be handled in #3508